tests/MessagePack.Tests/DynamicObjectResolverTests.cs

```diff
@@ -524,6 +601,49 @@ public DefaultValueStringKeyStructWithExplicitConstructor(int prop1)
             }
         }
 
+        [MessagePackObject(KeyPolicy.ImplicitCamelCasePropertyNames)]
+        public class DefaultValueStringCamelKeyClassWithoutExplicitConstructor
+        {
+            public const int Prop1Constant = 102;
+            public const int Prop2Constant = 2223;
+
+            public int Prop1 { get; set; } = Prop1Constant;
```

Comment:
Thanks for the tests. They'll be more useful if the property names had multiple uppercase letters in them so that we can verify that only the first letter is changed.